### PR TITLE
avoid invalid account error on create vesting account

### PR DESCRIPTION
**Pull request by mmsqe**
* this change is related to https://github.com/cosmos/cosmos-sdk/pull/12154/files
* for more info see https://github.com/crypto-org-chain/cronos/issues/471

Closes: #XXX

## Description

______

For contributor use:

- [ ] Targeted PR against correct branch (see [CONTRIBUTING.md](https://github.com/cosmos/cosmos-sdk/blob/master/CONTRIBUTING.md#pr-targeting))
- [ ] Linked to Github issue with discussion and accepted design OR link to spec that describes this work.
- [ ] Code follows the [module structure standards](https://github.com/cosmos/cosmos-sdk/blob/master/docs/building-modules/structure.md).
- [ ] Wrote unit and integration [tests](https://github.com/cosmos/cosmos-sdk/blob/master/CONTRIBUTING.md#testing)
- [ ] Updated relevant documentation (`docs/`) or specification (`x/<module>/spec/`)
- [ ] Added  relevant `godoc` [comments](https://blog.golang.org/godoc-documenting-go-code).
- [ ] Added a relevant changelog entry to the `Unreleased` section in `CHANGELOG.md`
- [ ] Re-reviewed `Files changed` in the Github PR explorer

______

For admin use:

- [ ] Added appropriate labels to PR (ex. `WIP`, `R4R`, `docs`, etc)
- [ ] Reviewers assigned
- [ ] Squashed all commits, uses message "Merge pull request #XYZ: [title]" ([coding standards](https://github.com/tendermint/coding/blob/master/README.md#merging-a-pr))

### CODE REVIEW

File: types/account.go [hunk 1]
```diff
@@ -52,6 +52,10 @@ func ProtoAccount() authtypes.AccountI {
 	}
 }
 
+func (acc EthAccount) GetBaseAccount() *authtypes.BaseAccount {
```
Comment (fedekunze) [inline types/account.go:55]: Can you add a comment on the function?

**Review by fedekunze — APPROVED** [review]
Ack, pending changelog entry

File: types/account.go (hunk 1 above)
Comment (alexanderbez) [reply types/account.go:55]: nit: godoc yeah 👍

**Review by alexanderbez — APPROVED** [review]

**Review by crypto-facs — APPROVED** [review]

File: CHANGELOG.md [hunk 2]
```diff
@@ -570,6 +570,7 @@ corresponding Ethereum API namespace:
 * (evm) [tharsis#496](https://github.com/cosmos/ethermint/pull/496) Fix bugs on `journal.revert` and `CommitStateDB.Copy`.
 * (types) [tharsis#480](https://github.com/cosmos/ethermint/pull/480) Update [BIP44](https://github.com/bitcoin/bips/blob/master/bip-0044.mediawiki) coin type to `60` to satisfy [EIP84](https://github.com/ethereum/EIPs/issues/84).
 * (types) [tharsis#513](https://github.com/cosmos/ethermint/pull/513) Fix simulated transaction bug that was causing a consensus error by unintentionally affecting the state.
+* (types) [tharsis#1112](https://github.com/cosmos/ethermint/pull/1112) Add `GetBaseAccount` to avoid invalid account error when create vesting account.
```
Comment (fedekunze) [inline CHANGELOG.md:573]: This needs to go under Unreleased, please update

**Review by fedekunze — CHANGES_REQUESTED** [review]

File: CHANGELOG.md (hunk 2 above)
Comment (mmsqe) [reply CHANGELOG.md]: updated, but not sure the merge sequence, but this change could wait https://github.com/cosmos/cosmos-sdk/pull/12154

**Review by fedekunze — APPROVED** [review]

**Pull request merged**